portfolio/src/main/webapp/script.js

```diff
@@ -32,11 +32,70 @@ function addRandomFact() {
   factContainer.innerText = funFact;
 }
 
+/**
+ * Run functions on load
+ */
+function runOnLoadFunctions(){
+    getRandomGreeting();
+    getComments();
+}
+
 /**
  * Fetches random greeting from the server and adds it to the DOM.
  */
 function getRandomGreeting() {
     fetch('/greeting').then(response => response.text()).then((greeting) => {
     document.getElementById('greeting-header').innerText = greeting;
     });
+}
+
+/**
+ * Fetches comments from the server and adds it to the DOM.
+ */
+function getComments() {
+  fetch('/comments').then(response => response.json()).then((comments) => {
+    // stats is an object, not a string, so we have to
+    // reference its fields to create HTML content
+
+    const commentContainer = document.getElementById('comments-container');
+    createCommentElement(commentContainer, comments);
+  });
+}
+
+/** Creates a <div> element containing comment. */
+function createCommentElement(commentContainer, comments) {
+    if (comments.length == 0){
+        return commentContainer;
+    }
+
+    var i;
```

Comment:
small nit: perhaps you can declare all of these in one line?

Reply:
Will do! Thanks for the advice